### PR TITLE
developer docs improvements

### PR DESCRIPTION
**Pull request by anuejn**
- 📃️ doc: clarify server/README.md
- 📃️ doc: hint to server deps in app/README.md
- ☝️ upgrade numpy; document missing wheels
- ⚠️ mute ffmpeg not found warning in backend

### CODE REVIEW

File: server/README.md [hunk 1]
```diff
@@ -15,7 +15,10 @@ This can either be run on the same host as the app or hosted somewhere else.
 ## Installation & Usage
 
 We use [poetry](https://python-poetry.org/) for dependency management.
-Either install poetry using your system package manager or by running
+Either install poetry using your system package manager or by running.
```
Comment (pajowu) [inline server/README.md:18]: I think the code for installing poetry should go after this line

**Review by pajowu — CHANGES_REQUESTED** [review]
Nice, just 1 small remark

**Comment by jgarplind** [comment]
Nice! I pulled down the branch and verified that all requested improvements are there. Docs are more accurate, and confusing warnings are muted. 🎉

**Review by pajowu — APPROVED** [review]

**Pull request merged**